### PR TITLE
Add license of treesaver, cc #5500

### PR DESCRIPTION
**Pull request by lcd78706**
Reference: https://github.com/Treesaver/treesaver

### CODE REVIEW

**Comment by lcd78706** [comment]
@kennynaoh 
Please review this PR. Thanks!

File: ajax/libs/treesaver/package.json [hunk 1]
```diff
@@ -10,5 +10,9 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/Treesaver/treesaver"
-  }
+  },
+  "licenses": [
+    "MIT",
+    "GPL-1.0"
```
Comment (kennynaoh) [inline ajax/libs/treesaver/package.json]: @lcd78706 
I think it's version 2 https://github.com/Treesaver/treesaver#license

**Comment by kennynaoh** [comment]
@lcd78706 Please ping me again after you fix the problem next time. Thanks!

**Comment by kennynaoh** [comment]
LGTM
ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**